ipapython/ipaldap.py

```diff
@@ -1245,11 +1245,10 @@ def make_filter_from_attr(
             return cls.combine_filters(flts, rules)
         elif value is not None:
             if isinstance(value, bytes):
-                if six.PY3:
-                    value = value.decode('raw_unicode_escape')
+                value = u''.join(
+                    u'\\{:02x}'.format(i) for i in six.iterbytes(value))
```

Comment:
This is not performance critical code, but OK (I like the empty initial string hack :+1:).